pages/blog/20191014-kolibri.md

```diff
@@ -0,0 +1,37 @@
+# Access Kolibri on treehouses using docker
+
+14, October 2019 • [irisb1701](https://github.com/irisb1701)
+
+---
+
+Kolibri is a platform used by our "competitor" organization called Learning Equality similar to what Planet is for Open Learning Exchange.
+
+Below are instructions on how to run Kolibri on treehouses:
+
+1. Plug your RPi in and ssh into treehouses
+1. If you don't already have docker installed, follow [these instructions](https://docs.docker.com/v17.09/engine/installation/) on their official page to install it on mac/windows
+1. Now, go to the [treehouses github](https://github.com/treehouses)
+1. Scroll down until you find the `kolibri` repository
+1. Once in the repo, scroll down to where the readme.md file is displayed
+1. Copy the link provided: `docker run -v /data/save/location:/root/.kolibri -p 8080:8080 treehouses/kolibri
+1. Replace /data/save/location with /srv/kolibri
```

Comment:
> Replace /data/save/location with /srv/kolibri

Should the directories be in code blocks?
"Replace `/data/save/location` with `/srv/kolibri`"